src/main/java/seedu/contax/logic/commands/DeletePersonCommand.java

```diff
@@ -18,8 +18,8 @@
     public static final String COMMAND_WORD = "deleteperson";
 
     public static final String MESSAGE_USAGE = "`" + COMMAND_WORD
-            + "`: **Deletes the person identified by the index number used in the displayed person list.**\n"
-            + "Parameters: *INDEX (must be a positive integer)*\n"
+            + "`: **Deletes the person identified by the index number used in the displayed person list.**"
+            + "\nParameters: *INDEX (must be a positive integer)*\n"
             + "Example: `" + COMMAND_WORD + " 1`";
 
     public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
```

Comment:
Should have a break line after the colon here.

Reply:
thanks for the catch, updated in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/248/commits/2b67c04764a53ad085721a659134752196fea472